src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Escaping.cs

```diff
@@ -32,7 +32,6 @@ internal static partial class JsonWriterHelper
             1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, // U+0070..U+007F
         };
 
-        private const string HexFormatString = "X4";
         private static readonly StandardFormat s_hexStandardFormat = new StandardFormat('X', 4);
```

Comment:
I think it's a little unfortunate that it's suggesting the inlining of the single-use const here; because the "before" code has the two X4s declared adjacently.

Reply:
It wasn't suggesting it... it was just flagging the const as being unused in some builds.  I can change it back and suppress it with a pragma.  But I also don't particularly like single-use consts like this, especially ones that are so far away from their use sites.

Reply:
Ah, the good old 6 of one, half-dozen of the other...

I, personally, like that the two X4s are together; but tests should catch it if they diverge, so I'm okay with it as is.